spec/factories/valid_variable_draft_factories.rb

```diff
@@ -53,8 +53,8 @@
     draft_type 'VariableDraft'
 
     draft {{
-      'Name': draft_short_name || "#{Faker::Space.galaxy}_#{Faker::Number.number(6)}",
-      'LongName': draft_entry_title || "#{Faker::Space.nebula} #{Faker::Space.star_cluster} #{Faker::Number.number(6)}",
+      'Name': draft_short_name || 'PNs_LIF',
```

Comment:
I think we want to leave the old way of doing this (generating random stuff) given the uniqueness constraint.